AccountService.js

```diff
@@ -54,25 +55,26 @@ export class AccountService {
 
   /**
    * Takes a url and an email then creates an account for a user.
-   * @memberOf AccountService
-   * @param {Object} options
-   * @param {string} [options.url = /accounts]
-   * @param {string} options.email
    *
-   * @return {Object}
+   * @param {object} options - The options to use.
+   * @param {string} [options.url = /accounts] - The base baseUrl.
+   * @param {string} options.email - The user's email.
+   *
+   * @returns {Promise<object>} - Data.
```

Comment:
```suggestion
   * @returns Promise<object> - Data.
```

Reply:
Here and throughout.

Reply:
Eh?  We have `{}` around return types everywhere else.  Why would they be removed?

Reply:
We do?  Why?

Reply:
If that's the case, I think it's a mistake as we started to use the `Promise` keyword.  JSDOCs in my opinion is one of the most poortly documented APIs I've ever seen.

Reply:
Yes, jsdoc needs better docs.  There is a bunch of basic info, but sometimes it's hard to figure out how to use it.  Some docs are better than no docs though.  Sometimes they have `{Promise}`, and sometimes they have `{Promise<...>}`.  I don't think we've been strict about using the `@async` tag.  Which you think would just imply the Promise part.
https://jsdoc.app/tags-returns.html
https://jsdoc.app/tags-async.html

Reply:
Should I remove the `Promise` keyword and change it back to how it was before?

Reply:
I think this shows exactly what we want: https://jsdoc.app/tags-async.html

Assuming that our jsdoc linting supports the 3.5 API.  Do we know?

So in addition to curly braces around Promise we should be using the @async tag.

Reply:
It may be that tools can infer from code that it's async.  Seems odd to have to duplicate that detail.

@aljones15 Do you happen to have advice on the best practices here?  I think we were going back on forth on these issues before.

Reply:
https://jsdoc.app/tags-async.html @mattcollier @davidlehn It looks like we don't really need to use the async tag.

Reply:
OK, we can discuss this further in another venue if people have interest.  Thanks for looking into it.